프로그래머스/1/82612. 부족한 금액 계산하기/부족한 금액 계산하기.js

```diff
@@ -0,0 +1,9 @@
+function solution(price, money, count) {
+  let sum = 0;
+  let arr = [];
+  for (let i = 1; i <= count; i++) {
+    arr.push(price * i);
+    sum = arr.reduce((a, b) => a + b);
+  }
+  return sum > money ? sum - money : 0;
```

Comment:
```suggestion
  const totalCost = price * count * (count + 1) / 2; 
  return Math.max(0, totalCost - money); 
```
엇 제가 도담님께도 말씀을 드렸는데 총 비용을 한 번만 계산한 후에 return 값을 음수가 되지 않도록 Math.max메서드를 사용하는것도 좋을것 같다구 생각했습니다 !!